src/StoreKeeper/WooCommerce/B2C/Frontend/Handlers/OrderHookHandler.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace StoreKeeper\WooCommerce\B2C\Frontend\Handlers;
+
+use StoreKeeper\WooCommerce\B2C\Factories\LoggerFactory;
+use StoreKeeper\WooCommerce\B2C\Helpers\HtmlEscape;
+use StoreKeeper\WooCommerce\B2C\I18N;
+use StoreKeeper\WooCommerce\B2C\Imports\OrderImport;
+
+class OrderHookHandler
+{
+    public const STOREKEEPER_ORDER_TRACK_HOOK = 'storekeeper_order_tracking_message';
+
+    public function addOrderStatusLink(\WC_Order $order): void
+    {
+        $storekeeperId = $order->get_meta('storekeeper_id', true);
+
+        $orderPageStatusUrl = null;
+
+        if (!empty($storekeeperId)) {
+            try {
+                $orderPageStatusUrl = OrderImport::ensureOrderStatusUrl($order, $storekeeperId);
+            } catch (\Throwable $throwable) {
+                LoggerFactory::create('order')->error($throwable->getMessage(), ['trace' => $throwable->getTrace()]);
+            }
+        }
+
+        if (!empty($orderPageStatusUrl)) {
+            $trackMessage = esc_html__('Your order is ready for track and trace.', I18N::DOMAIN);
```

Comment:
```suggestion
            $trackMessage = esc_html__('To check your parcel status go to the <a href=%">Track & Trace page </a>.', I18N::DOMAIN);
```
Also more the message to createOrderTrackingMessage, because it depends on the styling

Reply:
Can't move the message inside the `createOrderTrackingMessage` as it should be passed to the next calling filter.

E.g.:
1. Default filter got the message and the URL, it will generate the message with the `div storekeeeper-track-trace-box`.
2. Themers filter will receive the first parameter as `div storekeeeper-track-trace-box` + message from the default, so they can override it by return a new one. The URL parameter won't change during this workflow.